custom_converter/README.md

```diff
@@ -2,7 +2,7 @@
 
 This sample shows how to make a custom payload converter for a type not natively supported by Temporal.
 
-To run, first see [README.md](../README.md) for prerequisites. Then, run the following from this directory to start the
+To run, first see [README.md](../README) for prerequisites. Then, run the following from this directory to start the
```

Comment:
This breaks GitHub. For example, click the link in your branch: https://github.com/temporalio/samples-python/tree/TW-219-update-python-links/custom_converter

Reply:
The extension is still removed here